plans/upstream-keylime-containers.fmf

```diff
@@ -0,0 +1,33 @@
+summary:
+  Tests used by Packit/TFT CI on Github to test upstream keylime containers
+
+environment+:
+  TPM_BINARY_MEASUREMENTS: /var/tmp/binary_bios_measurements
+  KEYLIME_RUST_CODE_COVERAGE: 1
+  AGENT_DOCKERFILE: Dockerfile.upstream.c9s
+  REGISTRY: quay.io
+  VERIFIER_IMAGE: keylime/keylime_verifier:master
+  REGISTRAR_IMAGE: keylime/keylime_registrar:master
+  AGENT_IMAGE: keylime/keylime_agent:master
+
+discover:
+  how: fmf
+  test:
+   - /setup/configure_swtpm_device
+   - /setup/install_upstream_keylime
+   - /setup/install_rust_keylime_from_copr
+   # change IMA policy to simple and run one attestation scenario
+   # this is to utilize also a different parser
+   - /setup/configure_kernel_ima_module/ima_policy_simple
+   - "/container/.*"
+
+execute:
+    how: tmt
+
+adjust+:
+  - when: target_PR_branch is defined and target_PR_branch != main
+    enabled: false
+    because: we want to run this plan only for PRs targeting the main branch
+
+  - when: distro != fedora-37
```

Comment:
FYI, it is not possible to configure swtpm device on C9S. I am just updating the other plan in
https://github.com/RedHat-SP-Security/keylime-tests/pull/436

Reply:
Thank you, I'll fix this